backpack-lint/src/main/java/net/skyscanner/backpack/lint/check/BpkComposeComponentUsageDetector.kt

```diff
@@ -159,6 +159,14 @@ class BpkComposeComponentUsageDetector : Detector(), SourceCodeScanner {
         "androidx.compose.material.TextField",
       )
     ),
+    SLIDER(
+      fullName = "net.skyscanner.backpack.compose.slider.BpkSlider",
+      webName = "slider",
+      componentsToReplace = setOf(
+        "androidx.compose.material.Slider",
+        "androidx.compose.material.RangeSlider",
```

Comment:
for range slider the replacement isn't `BpkSlider`, but rather `BpkRangeSlider`. Might be better to have two separate entries here to map to the right replacement. The web name would stay the same, though as the docs are both in the same place

Reply:
Fixed this but left `full name` same as BpkSlider, as they are both in the same file. Please let me know if this needs fixing!